app/src/main/java/chat/rocket/android/api/MethodCallHelper.java

```diff
@@ -304,6 +307,19 @@ public MethodCallHelper(RealmHelper realmHelper, DDPClientWrapper ddpClient) {
         .onSuccessTask(task -> Task.forResult(null));
   }
 
+  public Task<Void> pushUpdate(@NonNull String pushId, @NonNull String token,
+                               @Nullable String userId) {
+    return call("raix:push-update", TIMEOUT_MS, () -> {
+      JSONObject param = new PushUpdate(pushId, token, userId).toJson();
+      return new JSONArray().put(param);
+    }).onSuccessTask(task -> Task.forResult(null));
+  }
+
+  public Task<Void> pushSetUser(String pushId) {
+    return call("raix:push-setuser", TIMEOUT_MS, () -> new JSONArray().put(pushId))
+        .onSuccessTask(task -> Task.forResult(null));
+  }
+
```

Comment:
It is better to define these method in another helper class as FileUploadingHelper does.